playtime/init.lua

```diff
@@ -53,15 +53,24 @@ local function format_duration(seconds)
 end
 
 minetest.register_chatcommand("playtime", {
-	params = "",
+	params = "<player_name>",
 	description = S("See your total playtime on this server"),
-	func = function(name)
-		if minetest.get_player_by_name(name) then
+	func = function(name, player_name)
+		if player_name == '' then
+            player_name = name
+        end
+		if (player_name == '') and minetest.is_singleplayer() then
+			player_name = 'singleplayer'
+		end
```

Comment:
```suggestion
```

This code is redundant, the previous line should do the same thing

Reply:
when I was testing locally if you ran /playername and didn't provide a username it would default to '' instead of 'singleplayer' and would not work right although that was for the old version that is on contentdb.

Reply:
If you don't provide a playername it should use `name`, which will be `singleplayer` if they're in a singleplayer world